d2core/d2asset/asset_manager.go

```diff
@@ -474,6 +477,99 @@ func (am *AssetManager) commandAssetClear([]string) error {
 	am.transforms.Clear()
 	am.animations.Clear()
 	am.fonts.Clear()
+	am.dt1s.Clear()
+	am.ds1s.Clear()
+	am.dccs.Clear()
+	am.cofs.Clear()
 
 	return nil
 }
+
+func (am *AssetManager) LoadDT1(dt1Path string) (*d2dt1.DT1, error) {
+	if dt1Value, found := am.dt1s.Retrieve(dt1Path); found {
+		return dt1Value.(*d2dt1.DT1), nil
+	}
+
+	fileData, err := am.LoadFile("/data/global/tiles/" + dt1Path)
```

Comment:
we may want to put these paths into `d2resource` at some point. I think there are a couple more used in the audio engine.